.travis.yml

```diff
@@ -7,6 +7,8 @@ script:
 - npm run code-check
 - npm run cover
 - grunt build
+before_deploy:
+- cd dist
```

Comment:
This should fix #8 where @jboerjes found that the deployed package hasn't been transpiled to ES5